SDWebImageAVIFCoder/Classes/Private/ColorSpace.h

```diff
@@ -0,0 +1,19 @@
+//
+//  ColorSpace.h
+//  SDWebImageAVIFCoder
+//
+//  Created by psi on 2020/03/15.
+//
+
+#pragma once
+#if __has_include(<libavif/avif.h>)
+#import <libavif/avif.h>
+#else
+#import "avif/avif.h"
+#endif
+
+CGColorSpaceRef CreateColorSpaceMono(avifNclxColourPrimaries const colorPrimaries, avifNclxTransferCharacteristics const transferCharacteristics);
```

Comment:
These C function share the global symbol, which may cause conflict. Can we add some prefix here, like `AVIFCreateColorSpaceMono` ?
And the visibility should be marked as `__attribute__((visibility("hidden")))`.

The same applies to others C function

Reply:
Thanks, I renamed functions and added `__attribute__((visibility("hidden")))`, `_Nonnull `, `_Nullable`.

Reply:
This final function name is `SDCreateAVIFCreateColorSpaceRGB`, note there are two `Create` 🤣 

Can this be `SDCreateAVIFColorSpaceRGB` ?

Reply:
Oh! :laughing:
I missed two "Create" (it might be too long function names for me!)

I renamed the functions.

I see... thanks. > Core Foundation Naming